src/SDKs/Monitor/Monitor.Tests/Monitor.Tests.csproj

```diff
@@ -8,6 +8,10 @@
 	</PropertyGroup>
 	<PropertyGroup>
     <TargetFrameworks>net452;netcoreapp1.1</TargetFrameworks>
+  </PropertyGroup>
+	<PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net452|AnyCPU'">
+	  <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
+	  <WarningsAsErrors />
```

Comment:
Please remove these settings since these are already imported from the common targets files